### PR TITLE
Persons in f.34

### PR DESCRIPTION
**Pull request by MassimoVilla**
Two individuals involved in the history of the ms

### CODE REVIEW

**Review by thea-m — APPROVED** [review]

File: new/PRS13275HaylaGi.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ኀይለ፡ ጊዮርጊስ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Ḫayla Giyorgis</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic">ʾAbbā</occupation>
```
Comment (SusanneHummel) [inline new/PRS13275HaylaGi.xml:51]: "ʾAbbā" is an occupation? Not a title?

Maybe "monk"? or better to state nothing in occupation if we don't know for sure?

File: new/PRS13276Thyne.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2020-12-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="en" xml:id="n1">W.K. Thyne</persName>
```
Comment (SusanneHummel) [inline new/PRS13276Thyne.xml:48]: Did you find any more information on him in the world wide web?

**Review by SusanneHummel — CHANGES_REQUESTED** [review]

File: new/PRS13275HaylaGi.xml (hunk 1 above)
Comment (MassimoVilla) [reply new/PRS13275HaylaGi.xml:51]: I have removed ʾAbbā from there and added that roleName within   `<persName type="alt"/>`, according to an example provided in the guidelines

File: new/PRS13276Thyne.xml (hunk 2 above)
Comment (MassimoVilla) [reply new/PRS13276Thyne.xml:48]: He is seemingly quoted in some commercial reports as one of the officers of the Royal Indian Marine in the second half of the 19th cent. and in the first years of the 20th century. But I haven't found more than this.

**Review by SusanneHummel — APPROVED** [review]

File: new/PRS13276Thyne.xml (hunk 2 above)
Comment (Gnisci) [reply new/PRS13276Thyne.xml:48]: I think this is fine, to trace the history of all this people would be a project within the project. Interesting, but not necessary!

**Pull request merged**